Solutions/Azure Batch Account/SolutionMetadata.json

```diff
@@ -0,0 +1,15 @@
+{
+	"publisherId": "azuresentinel",
+	"offerId": "azure-sentinel-solution-batchaccount-preview",
+	"firstPublishDate": "2021-11-23",
```

Comment:
Hey @v-laanjana, please change first publish date to today's date or PR raised date.